.github/workflows/reviewdog.yml

```diff
@@ -8,6 +8,26 @@ permissions:
   checks: write
 
 jobs:
+  rubocop:
+    name: Rubocop
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up Ruby and run bundle install
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: rubocop
+        uses: reviewdog/action-rubocop@e70b014b8062c447d6b515ee0209f834ea93e696
+        with:
+          use_bundler: true
```

Comment:
I am not sure about this, but will adding a `skip_install: true` be an improvement according to a suggestion [here](https://github.com/reviewdog/action-rubocop/issues/83)?

Reply:
I tried this out on my sandbox repo, and it looks like it worked fine with this enabled!